enginetest/script_queries.go

```diff
@@ -456,6 +456,52 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "Test cases on select into statement",
+		SetUpScript: []string{
+			"SELECT 1 INTO @abc",
+			"SELECT * FROM (VALUES ROW(22,44,88)) AS t INTO @x,@y,@z",
+			"CREATE TABLE tab1 (id int primary key, v1 int)",
+			"INSERT INTO tab1 VALUES (1, 1), (2, 3), (3, 6)",
+			"SELECT id FROM tab1 ORDER BY id DESC LIMIT 1 INTO @myVar",
+			"CREATE TABLE tab2 (i2 int primary key, s text)",
+			"INSERT INTO tab2 VALUES (1, 'b'), (2, 'm'), (3, 'g')",
+			"SELECT m.id, t.s FROM tab1 m JOIN tab2 t on m.id = t.i2 ORDER BY t.s DESC LIMIT 1 INTO @myId, @myText",
+			// TODO: union statement does not handle order by and limit clauses
+			//"SELECT id FROM tab1 UNION select s FROM tab2 LIMIT 1 INTO @myUnion",
```

Comment:
I don't think this is something MySQL supports either, right? The `order by` and `limit` clauses attach to the second select statement, and don't work globally on the unioned results.

Reply:
MySQL allows order by and limit around multiple unions. 
```
mysql> select * from (VALUES ROW(2), ROW(15), ROW(4)) as t1 union select * from (VALUES ROW(7), ROW(3), ROW(9)) as t2 ORDER BY column_0 DESC LIMIT 1;
+----------+
| column_0 |
+----------+
|       15 |
+----------+
```

Reply:
Nice! You're totally right.